x-pack/test/functional/apps/discover/reporting.ts

```diff
@@ -112,29 +116,29 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         // click 'Copy POST URL'
         await PageObjects.share.clickShareTopNavButton();
         await PageObjects.reporting.openCsvReportingPanel();
-        const advOpt = await find.byXPath(`//button[descendant::*[text()='Advanced options']]`);
-        await advOpt.click();
-        const postUrl = await find.byXPath(`//button[descendant::*[text()='Copy POST URL']]`);
-        await postUrl.click();
+        await testSubjects.existOrFail('shareReportingCopyURL');
+        await testSubjects.click('shareReportingCopyURL');
+        await PageObjects.share.closeShareModal();
 
         // get clipboard value using field search input, since
         // 'browser.getClipboardValue()' doesn't work, due to permissions
         const textInput = await testSubjects.find('fieldListFiltersFieldSearch');
         await textInput.click();
-        await browser.getActions().keyDown(Key.CONTROL).perform();
+        await browser
+          .getActions()
+          .keyDown(process.platform === 'darwin' ? Key.COMMAND : Key.CONTROL)
+          .perform();
         await browser.getActions().keyDown('v').perform();
 
         const reportURL = decodeURIComponent(await textInput.getAttribute('value'));
-
         // get number of filters in URLs
         const timeFiltersNumberInReportURL =
-          reportURL.split('query:(range:(order_date:(format:strict_date_optional_time').length - 1;
+          reportURL.split(/query:\(range:\(order_date:\(format:strict_date_optional_time/).length -
+          1;
         const timeFiltersNumberInSharedURL = sharedURL.split('time:').length - 1;
-
-        expect(timeFiltersNumberInSharedURL).to.be(1);
+        expect(timeFiltersNumberInSharedURL).equal(timeFiltersNumberInReportURL);
```

Comment:
Are these no longer showing up as `1`? Do you happen to know what they're showing up as now?

Reply:
I saw that both were 1 and just made the check to make sure they matched - let me put this back, thank you!! [24219cc](https://github.com/elastic/kibana/pull/171253/commits/24219ccf149deefdd80cc01d35ee74d1259d807c)